packages/block-library/src/query-no-results/block.json

```diff
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "core/query-no-results",
+	"title": "No results",
+	"category": "theme",
+	"description": "Contains the block elements used to render content when no query results are found.",
```

Comment:
I have to spend a few seconds re-reading this description in order to understand what I need to do, so IMO it's sub-optimal.
However, it is consistent with the comment-template and post-template blocks which use similar wording, so if this changes it would ideally also change in other blocks as well - so not something specific to this PR.

Reply:
I like the first suggestion, but I would like the related block descriptions to have a "simpler" language, and not only this new block.